docs/OnTrack/Jupyter Notebook/Docker-Documentation-Research-T1-2022.md

```diff
@@ -1,32 +1,67 @@
 # Docker Documentation and Research (WIP) T1-2022
 
-- [Back to Jupyter Notebook Documentation Index](Index.md)
+[Back to Jupyter Notebook Documentation Index](Index.md)
 
-## Conversion Relating to Docker Processes
+## Intro
+
+As we know, the Jupyter Notebook conversion feature will happen via processes that occur inside
+Docker containers. In this document we will discuss the main architecture of this structure in
```

Comment:
I would remove the "as we know". Assume this documentation is being read by people who have never worked on this project before and they don't know anything